.github/workflows/validation-check.yaml

```diff
@@ -10,7 +10,26 @@ jobs:
     if: startsWith(github.event.pull_request.base.ref, 'dev-v') || startsWith(github.event.pull_request.base.ref, 'release-v')
     runs-on: ubuntu-latest
     steps:
+      - name: Check for modifications in specified folders
+        id: check-modifications
+        uses: actions/github-script@v4
```

Comment:
@nicholasSUSE Is there any reason to use an [old version](https://github.com/marketplace/actions/github-script) of this action? In fact, this version is raising some warnings in the log messages.

Reply:
@recena 
Forgive me I did not see that before. 
Looking on Warnings, other software need updates:
```
Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: actions/checkout@v3.
```

I will update them all on these PRs.